Sources/Sentry/include/HybridPublic/SentryEnvelope.h

```diff
@@ -8,8 +8,12 @@
 
 #endif
 
-@class SentryEvent, SentrySession, SentryId, SentryUserFeedback, SentryAttachment,
-    SentryTransaction, SentryClientReport, SentryEnvelopeItemHeader;
+@class SentryEvent;
```

Comment:
I removed `SentryTransaction` and `SentryClientReport`.

Reply:
Thank you for also splitting it into individual declarations 🙏🏻